stix2validator/__init__.py

```diff
@@ -1,9 +1,14 @@
 # Expose certain functions and classes to the stix2validator namespace
 # flake8: noqa
 
+import logging
+import sys
+
 from .errors import ValidationError
 from .output import print_results
 from .util import ValidationOptions
 from .validator import (run_validation, validate, validate_file,
                         validate_parsed_json, validate_string)
 from .version import __version__
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO, format='%(message)s')
```

Comment:
This is probably OK, because `basicConfig()` won't do anything if logging is already configured, but ideally this would only be done in the command-line script.